src/components/buttons/RadioButton/RenderOptions.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
```

Comment:
I think you can add this inside RadioButtonGroup.tsx but outside RadioButtonGroup, so:

```
const Options = () => {};
export const RadioButtonGroup = () => {};
```